shared/src/test/diff/mlscript/BadAlias.mls

```diff
@@ -14,7 +14,7 @@ type A = A
 
 res: "oops"
 //│ res: "oops"
-//│    = Uncaught ReferenceError: res is not defined
+//│    = undefined
```

Comment:
Why `undefined`?

Reply:
🧐 Because previous block is marked as `:e`.

> Maybe we should still execute code that does not type check, but just expect and ignore any resulting runtime errors.

Yes, you’re right.

Reply:
Oh right, I see.

Reply:
Changed since ed2b155: even marked as `:e`, test cases will be executed anyway.